auter.yumdnfModule

```diff
@@ -113,6 +113,7 @@ function apply_updates() {
 
     local TRANSACTIONID=$(${PACKAGE_MANAGER} history info 2>&1 | grep "Transaction ID")
     logit "INFO: Updates complete (${PACKAGE_MANAGER} ${TRANSACTIONID}). You may need to reboot for some updates to take effect"
+    log_last_run
```

Comment:
Ok that seems reasonable for the most part. I do think that its a valid log to be added if the apply  function completes successfully when there are no updates. This implies that auter did attempt to update the system and there were no errors. So we should add another call to the log_last_run function in the following elif block.